sdk/ml/azure-ai-ml/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "python",
   "TagPrefix": "python/ml/azure-ai-ml",
-  "Tag": "python/ml/azure-ai-ml_d05969c2b4"
+  "Tag": "python/ml/azure-ai-ml_b8d9d78843"
```

Comment:
how doe the new recording process take for your PR?

Reply:
there is a separate document for this. generally speaking, we need to run `python .\scripts\manage_recordings.py push -p sdk/ml/azure-ai-ml` under repo root to push our recording change